web-registry/src/lib/ga.ts

```diff
@@ -1,105 +0,0 @@
-import { useEffect, useState } from 'react';
```

Comment:
this file is no longer needed with the new analytics integration: https://github.com/regen-network/regen-web/pull/1188/files#r988437204